app/assets/javascripts/fae/form/_validator.js

```diff
@@ -26,10 +26,13 @@ Fae.form.validator = {
   /**
    * Validate the entire form on submit and stop it if the form is invalid
    */
-  formValidate: function () {
+  formValidate: function ($scope) {
     var _this = this;
 
-    FCH.$document.on('submit', 'form:not([data-remote=true])', function (e) {
+    if (typeof($scope) === 'undefined'){
```

Comment:
FWIW Frob Core has a [`setDefault`](https://github.com/wearefine/fae/blob/master/app/assets/javascripts/fae/vendor/frob_core_helpers.js#L317-L319) for better legibility